cmd/alertmanager/main.go

```diff
@@ -92,6 +101,54 @@ func instrumentHandler(handlerName string, handler http.HandlerFunc) http.Handle
 
 const defaultClusterAddr = "0.0.0.0:9094"
 
+// buildReceiverIntegrations builds a list of integration notifiers off of a
+// receiver config.
+func buildReceiverIntegrations(nc *config.Receiver, tmpl *template.Template, logger log.Logger) ([]notify.Integration, error) {
```

Comment:
Shouldn't this be inside the `github.com/prometheus/alertmanager/notify` package?

Reply:
Depending on @simonpasquier's reasoning, I could understand it being moved here. It seems like this is a bunch of imperative code using the notify package components, so it's kind of the glue tying together a library? :man_shrugging:, will wait to see what he has to say.

Reply:
I saw no concrete reason to keep it in `github.com/prometheus/alertmanager/notify` as it is the glue between configuration and notifiers.

Reply:
:+1: